### ISSUE TITLE
PHS Reachout / CFJ (Please see callforjustice.github.io for more information) / Condemnation / Request for Sacrificial Regards

### ISSUE THREAD

**Issue by Bobby2552** [AnZaNaMa/technika#1]
# Dear Mr.AnZaNamA,

As a member of the board of Pokemon Humane Society, I am forced to condone your recent endorsement of the competitive Pokemon League Sport (PLS for short (Please see pls.io for more information)) as an inexcusably cruel and Nazi-like gesture that requires your immediate apology and ritual sacrifice for the betterment of the community.  Please RSVP for your sacrifice, we are preparing the sacred blade and the pits of fire as the message is being transmitted over the Internet Superhighway (Please see internetsuperhighway.org for more information).

Yours truly,
## ~~ xXx_PHS-MaSt35-6969696969_xXx ~~
> "When it doubt, jack it out" - Marvin the Molester

**Comment by AnZaNaMa** [comment]
I think you must be confused. My post was about how inhumane the fighting of Pokémon is. Also how did you find my github?

**Comment by Bobby2552** [comment]
Roomate searched your name on Google and found the second google image of your name, and it was linked to your GitHub lol. He had me post this cause I've got balls lol. Sorry to freak you out haha. Best of luck on the test :)

**Comment by AnZaNaMa** [comment]
Lol okay. I put way too much stuff online and use the same username for everything so I imagine it must be pretty simple to find me. I just thought it was a little spooky that you posted in a repo I made yesterday and haven't uploaded anything to yet lol

**Comment by Bobby2552** [comment]
Haha yeah it was the first one on the list. What will the mod do if I may ask?

**Comment by AnZaNaMa** [comment]
Mainly just add in machines and stuff maybe item storage, regular tech mod stuff. But instead of using some sort of generator system with electricity, it'll have some sort of magical aspect. Haven't fully figured it out yet. Just thought of it yesterday :P

**Comment by Bobby2552** [comment]
Oh sweet good idea. I've got a couple servers running, and one of the big things I would want in a mod is a decent sorting system. I know Tekkit has a few solutions, but something less known is more fun.

**Comment by AnZaNaMa** [comment]
Yeah, it'll probably take me a while to finish it though. Mainly just because I'm really bad at rendering

**Comment by Bobby2552** [comment]
What do you mean by rendering? What is there to render?

**Comment by AnZaNaMa** [comment]
The entire game is rendered. It's not like you're just looking at a text box when you play minecraft
